proto/iotics/api/twin.proto

```diff
@@ -275,7 +275,8 @@ message UpsertTwinRequest {
     // Unique ID of the twin to create/update
     string twinId = 1;
 
-    // Twin visibility. Default value: 'PRIVATE'
+    // DEPRECATED - see field description for details.
+    // Default value: 'PRIVATE'
```

Comment:
Here we might also need to say that:

if any metadata network allowlist is provided in the props, the visibility value will be ignored